webapps-schema/src/main/java/io/camunda/webapps/schema/descriptors/ImportValueTypes.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.webapps.schema.descriptors;
+
+public enum ImportValueTypes {
```

Comment:
💭  shall we rename this to `ExportValueType` and reuse it in the `CamundaExporterRecordFilter` instead of the set defined there?

💭 I might be wrong, but I think across the camunda codebase, we use singular rather than plural enum names

Reply:
With regards to the first point I don't think so, specifically this refers to all the value types the operate and tasklist importers can handle and we only track if those ones are finished. For example the importers don't handle any `USER` type.